templates/admin/user/edit.tmpl

```diff
@@ -109,6 +109,12 @@
 					</div>
 				</div>
 				{{end}}
+				<div class="inline field">
```

Comment:
I'd place some divider here, to indicate that it is different UI element from rest of checkboxes.

Reply:
Also I wonder if we could just not display this if user is not 2FA enrolled, we already fetch the state for a list so it should be simple check in template.

Reply:
no it's not a simple template change ... but I'll add the check